REFERENCE.md

```diff
@@ -694,6 +696,27 @@ Returns a promise that resolves or rejects when the job completes or fails.
 
 ---
 
+### Job#moveToCompleted
+
+```ts
+moveToCompleted(returnValue, ignoreLock): Promise<string[]>
+```
+
+Moves a job to the completed queue. Pulls a job from 'waiting' to 'active' and returns a promise with it's id.
+
```

Comment:
It actually does not return an Id, but a tuple containing the next job data and the next job id (as commented above) or null, if no more jobs left in the waiting list. So maybe we should declare it like this:
```typescript
moveToCompleted(returnValue, ignoreLock): Promise<[JobData, JobId] | null>
```

Reply:
👍